src/OpenTelemetry.Exporter.Console/ConsoleMetricExporter.cs

```diff
@@ -80,7 +80,7 @@ public override ExportResult Export(in Batch<Metric> batch)
                 {
                     string valueDisplay = string.Empty;
                     StringBuilder tagsBuilder = new StringBuilder();
-                    for (int i = 0; i < metricPoint.Keys.Length; i++)
+                    for (int i = 0; i < ((metricPoint.Keys == null) ? 0 : metricPoint.Keys.Length); i++)
```

Comment:
Looks like this will make the check in every loop.
Consider extracting the null check.

Reply:
I command all the compilers and JITs in the world, to read the spec and code fully, and optimize this for me ;)

Reply:
https://i.pinimg.com/564x/7b/e7/08/7be708f65a78b298dd38e01b172cc6fe.jpg